registrations/tasks.py

```diff
@@ -314,10 +312,7 @@ def create_popi_subscriptionrequest(self, registration):
             return "POPI Subscription request not created"
 
         self.log.info("Fetching messageset")
-        msgset_short_name = utils.get_messageset_short_name(
-            "popi", registration.source.authority, None
-        )
-        r = utils.get_messageset_schedule_sequence(msgset_short_name, None)
+        r = ""
```

Comment:
Same here, we can just delete this whole task once we're not using it, instead of changing functionality

Reply:
This is same as above, the task is needed by another `validate_implement`, I have tried removing it and the entire validate fails, so I opted for removing the part that is not needed in the task

Reply:
Ah okay, I see, but I think all of the old `validate_implement`s are no longer needed, so we can remove all of them.

Reply:
okay, that will help a lot, thank you